Octokit/Clients/RepositoriesClient.cs

```diff
@@ -502,7 +521,8 @@ public Task<Branch> GetBranch(string owner, string repositoryName, string branch
             Ensure.ArgumentNotNullOrEmptyString(repositoryName, "repositoryName");
             Ensure.ArgumentNotNullOrEmptyString(branchName, "branchName");
 
-            return ApiConnection.Get<Branch>(ApiUrls.RepoBranch(owner, repositoryName, branchName));
+            const string previewAcceptsHeader = "application/vnd.github.loki-preview+json";
```

Comment:
This `const string` is duplicated in 3 methods. Could we declare it at the class level and use it?